tools/building.py

```diff
@@ -608,7 +608,7 @@ def move_to_safe_7bit_ascii_filename(filename):
 
   # Specify output file relative to the temp directory to avoid specifying non-7-bit-ASCII path names.
   cmd += ['--js_output_file', os.path.relpath(outfile, tempfiles.tmpdir)]
-  if pretty:
+  if settings.MINIFY_WHITESPACE:
```

Comment:
Now this looks flipped?

Reply:
oops.  Fixed.  Kind of worrying is this isn't covered by tests :-/  (I didn't let them all run so perhaps they are).

Reply:
They should be covered - I would be very surprised if they were not.

Reply:
CI should be fixed now.